tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/runechat.tsx

```diff
@@ -0,0 +1,11 @@
+import { FeatureColorInput, FeatureToggle, Feature, CheckboxInput } from "../../base";
+
+export const enable_chat_color_player: FeatureToggle = {
+  name: "Runechat color",
```

Comment:
I'd rename these both to `Custom Runechat Color`

Reply:
It depends, the character pref lists are sorted alphabetically, it's currently at the bottom (which I like), and adding Custom will bump it to the top.